src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/model/Prisoner.kt

```diff
@@ -328,6 +328,61 @@ open class Prisoner : Diffable<Prisoner> {
   @DiffableProperty(DiffCategory.PHYSICAL_DETAILS)
   var weightKilograms: Int? = null
 
+  @Schema(
+    description = "Hair colour. From PROFILE_CODES table where PROFILE_TYPE = HAIR. Allowable values extracted 07/02/2023.",
+    example = "Blonde",
+    allowableValues = ["Bald", "Balding", "Black", "Blonde", "Brown", "Brunette", "Dark", "Dyed", "Ginger", "Grey", "Light", "Mouse", "Multi-coloured", "Red", "White"]
+  )
+  @DiffableProperty(DiffCategory.PHYSICAL_DETAILS)
+  var hairColour: String? = null
```

Comment:
not sure if I should have put these in a physical characteristics object within the prisoner.  We don't seem to do this for any other fields - e.g. physical attributes / sentence details so have left as top level.